src/ui/public/doc_viewer/viewers/json.js

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash';
+import angular from 'angular';
+import 'ace';
+import docTableDetailViewsRegistry from 'ui/registry/doc_table_detail_views';
+
+import jsonHtml from './json.html';
+
+docTableDetailViewsRegistry.register(function () {
+  return {
+    title: 'JSON',
+    order: 20,
+    template: jsonHtml,
+    shouldShow: () => true,
```

Comment:
This is an interesting addition, might want to make `_.constant(true)` the default.